### PR TITLE
Add fdiaz as a member

### PR DESCRIPTION
**Pull request by fdiaz**
Adding myself as secondary for Airbnb after talking with @shepting

### CODE REVIEW

**Review by shepting — APPROVED** [review]
Thanks!

File: CONTRIBUTING [hunk 1]
```diff
@@ -3,7 +3,8 @@ Scott Berrevoets @sberrevoets (Lyft, secondary) - Until 2/2023
 Patrick Balestra @BalestraPatrick (Spotify) - Until 2/2023
 Dan Fleming @dflems (Spotify, Secondary) - Until 2/2023
 Steven Hepting @stevenhepting (Airbnb) - Until 2/2023
+Francisco Díaz @fdiaz (Airbnb) - Until 2/2023
```
Comment (shepting) [inline CONTRIBUTING]: You should likely add `, Secondary` in the brackets.

Comment (fdiaz) [reply CONTRIBUTING]: Updated! https://github.com/MobileNativeFoundation/foundation/pull/8/commits/e9168011a4de0ba1a0cad4cdc3743aeef841f7c1

**Comment by keith** [comment]
thanks! can you add the DCO here (details from CI should cover how)

**Comment by fdiaz** [comment]
Done! @keith

**Comment by keith** [comment]
Thanks!

**Pull request merged**